packages/playwright-test/src/mount.ts

```diff
@@ -23,6 +23,54 @@ export async function mount(page: Page, jsxOrType: any, options: any, baseURL: s
   }, true);
 }
 
+function serialize(value: any, visited: Set<any> = new Set()): any {
+  if (visited.has(value))
+    return undefined; // we found a cycle
+  if (typeof value === 'symbol')
+    return value;
+  if (Object.is(value, undefined))
+    return value;
+  if (Object.is(value, null))
+    return value;
+  if (Object.is(value, NaN))
+    return value;
+  if (Object.is(value, Infinity))
+    return value;
+  if (Object.is(value, -Infinity))
+    return value;
+  if (Object.is(value, -0))
+    return value;
+  if (typeof value === 'boolean')
+    return value;
+  if (typeof value === 'number')
+    return value;
+  if (typeof value === 'string')
+    return value
+  if (value instanceof Error)
+    return value;
+  if (value instanceof Date)
+    return value;
+  if (value instanceof RegExp)
+    return value;
```

Comment:
My brain was tired…most of this function could actually be squashed down if we end up needing it.